plutus-core/plutus-core/src/PlutusCore/Crypto/Hash.hs

```diff
@@ -1,13 +1,16 @@
 -- | Hash functions for lazy [[Data.ByteString.ByteString]]s
 {-# LANGUAGE TypeApplications #-}
-module Data.ByteString.Hash
```

Comment:
I decided to rename this because the other crypto stuff is in this directory already.